schema/event-origin.json

```diff
@@ -0,0 +1,13 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "EventOrigin",
+    "type": "object",
+    "additionalProperties": false,
+    "required": ["screen"],
+    "properties": {
+        "screen": {
+            "type": "string",
+            "enum": ["primaryScreen", "breakageForm"]
+        }
+    }
+}
```

Comment:
for now this just supports screen = "primaryScreen" or screen = "breakageForm"

This is how we'll decorate events so that the native side can decide what to do (eg: fire pixels etc)